assembly/src/parsers/serde/mod.rs

```diff
@@ -108,156 +108,158 @@ pub enum OpCode {
     U32CheckedRotlImm = 89,
     U32UncheckedRotl = 90,
     U32UncheckedRotlImm = 91,
-    U32CheckedEq = 92,
-    U32CheckedEqImm = 93,
-    U32CheckedNeq = 94,
-    U32CheckedNeqImm = 95,
-    U32CheckedLt = 96,
-    U32UncheckedLt = 97,
-    U32CheckedLte = 98,
-    U32UncheckedLte = 99,
-    U32CheckedGt = 100,
-    U32UncheckedGt = 101,
-    U32CheckedGte = 102,
-    U32UncheckedGte = 103,
-    U32CheckedMin = 104,
-    U32UncheckedMin = 105,
-    U32CheckedMax = 106,
-    U32UncheckedMax = 107,
+    U32CheckedPopcnt = 92,
```

Comment:
Just curious, why do we renumber everything?

Reply:
I think sequential numbering keeps it a bit cleaner - plus, it's not a lot of work to re-number :)